instance/management/commands/instance_redeploy.py

```diff
@@ -285,77 +297,54 @@ def _do_redeployment(self):
         update = self.options.get('update', {})
         sleep_seconds = self.options['batch_frequency']
 
-        # Loop termination is handled at the end of the loop
+        # Loop termination is handled at the end.
         while True:
-
-            # 1. Activate successful instances
-            reconfigure_load_balancer = False
+            # 1. Log successful instances
             for instance in self.success_tag.openedxinstance_set.iterator():
                 LOG.info("SUCCESS: %s [%s]", instance, instance.id)
                 instance.tags.remove(self.ongoing_tag)
                 instance.tags.remove(self.success_tag)
 
-                # Make sure that the latest appserver is running
+                # 1.1 Log completed instances
```

Comment:
I actually think the `completed` tag may now be irrelevant. I've delegated the logic that makes the appservers active into `spawn_appserver`, which also tags instances as failed/successful depending on whether they were actually successfully provisioned. We were marking things with an *extra* tag, namely `complete`, for a reason I'm quite uncertain of... I guess it may just have marked a way for us to see that, yes, we successfully made it past the logic that makes the appserver active. But now we don't need that because I modified `spawn_appserver` to make the appserver active upon success, and deactivate old ones.

So the work here should be to fix the potentially misleading docstrings, and maybe explain a little more what's happening.

Reply:
Ah, good - I was actually thinking of suggesting removing the `completed` tag, so I'm glad you went with that option.